.gitignore

```
@@ -1,6 +1,7 @@
 .env
 node_modules/
 db/schema.sql
+*.csv
```

```suggestion
csv/*.csv
```
경로가 다르기 때문에 명확하게 지정해야하고, 추가로 `git cache` 까지 제거해주셔야 git 으로 관리 되지 않습니다. push 후에는 항상  PR의 `Files changed` 탭을 확인하여 셀프체크 해주세요.